chromium_src/chrome/browser/ui/webui/settings/site_settings_helper.cc

```diff
@@ -61,15 +69,17 @@ base::StringPiece ContentSettingsTypeToGroupName(ContentSettingsType type) {
     return "ethereum";
   if (type == ContentSettingsType::BRAVE_SOLANA)
     return "solana";
+  if (type == ContentSettingsType::BRAVE_SHIELDS)
+    return "shields";
+  if (type == ContentSettingsType::BRAVE_HTTP_UPGRADABLE_RESOURCES)
+    return "shields_https";
```

Comment:
shouldn't this be `kHTTPUpgradableResources` ? And `kBraveShields` for shields above

Reply:
done, thanks!